src/components/CartPopover/CartPopover.js

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { inject, observer } from "mobx-react";
+import Grid from "@material-ui/core/Grid";
+import Hidden from "@material-ui/core/Hidden";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import CheckoutButtons from "components/CheckoutButtons";
+
+
+const styles = (theme) => ({
+  container: {
+    "alignItems": "center",
+    "backgroundColor": theme.palette.reaction.white,
+    "boxShadow": theme.boxShadow.depth2,
+    "display": "flex",
+    "marginLeft": "auto",
+    "marginRight": "auto",
+    "maxWidth": "400px",
+    "paddingTop": "12px",
+    "position": "fixed",
+    "right": 0,
+    "top": 0,
+    "transitionDuration": "400ms",
+    "transitionProperty": "transform",
+    "transitionTimingFunction": "linear",
+    "zIndex": theme.zIndex.appBar + 1,
+    "&:hover": {
+      transform: "translate(0px, 0px)"
+    }
+  },
+  gridContainer: {
+    padding: "10px"
+  },
+  isContainerHidden: {
+    transform: "translate(400px, 0px)"
+  },
+  isContainerVisible: {
+    transform: "translate(0px, 0px)"
+  },
+  containerItem: {
+    alignItems: "center",
+    display: "flex"
+  },
+  addedToCartImg: {
+    height: "40px",
+    marginRight: "10px",
+    width: "40px"
+  },
+  addedToCartItemName: {
+    maxWidth: "200px",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    fontWeight: theme.typography.fontWeightMedium,
+    display: "inline-block",
+    lineHeight: "0.8em"
+  },
+  addedToCartText: {
+    color: theme.palette.primary.dark,
+    display: "inline",
+    fontSize: theme.typography.fontSize * 0.875
+  }
+});
+
+@withStyles(styles, { withTheme: true })
+@inject("uiStore")
+@observer
+class CartPopover extends Component {
+  static propTypes = {
+    /**
+     * CartItem data
+     */
+    cartItem: PropTypes.shape({
```

Comment:
I'm seeing a warning in the browser console about this prop being undefined.
<img width="847" alt="screen shot 2018-07-11 at 8 24 21 am" src="https://user-images.githubusercontent.com/1135948/42575609-9755a3ce-84e6-11e8-8068-ba6627f79d17.png">